### DISCUSSION TITLE
Preconditioning not working

### DISCUSSION THREAD

**Discussion by larronni** [flobz/psa_car_controller#916] [Q&A]
Trying to get the preconditioning to work... But not sure what I am doing wrong here...
First, I send the command "HTTP://URL..../preconditioning/MYVIN/1" - in the browser, I get the response "true" back.
In the logs I get:
`2024-07-14 10:15:25,382 :: INFO :: Preconditioning: topic: psa/RemoteServices/from/cid/AC-ACNT200009312786/ThermalPrecond: {'asap': 'activate', 'programs': {'program1': {'day': [0, 0, 0, 0, 0, 0, 0], 'hour': 34, 'minute': 7, 'on': 0}, 'program2': {'day': [0, 0, 0, 0, 0, 0, 0], 'hour': 34, 'minute': 7, 'on': 0}, 'program3': {'day': [0, 0, 0, 0, 0, 0, 0], 'hour': 34, 'minute': 7, 'on': 0}, 'program4': {'day': [0, 0, 0, 0, 0, 0, 0], 'hour': 34, 'minute': 7, 'on': 0}}}`

The car does not start any preconditioning, and the app on the phone also says it is disabled.
(If I try to start it in the app, it works...)

If I check the status (run the "/get_vehicleinfo/") it looks like the command has not been sent to the car  (based on the date/time stamp)..?
```
  "preconditionning": {
    "air_conditioning": {
      "failure_cause": null,
      "status": "Disabled",
      "updated_at": "2024-07-13 20:07:03+00:00",
      "created_at": "2024-07-13 20:07:03+00:00"
    }

```
Does anyone know what could be wrong here...?

**Comment by fcassata** [comment]
Same here, running this on home assistant, can only get info from the car, not send anything to it.
